### PR TITLE
adds link-url

### PR DESCRIPTION
**Pull request by timothypratley**
Proposal: Allow //example.com to be a valid url (no http(s): in front of it) #9

### CODE REVIEW

File: src/cljx/jkkramer/verily.cljx [hunk 1]
```diff
@@ -134,10 +134,9 @@
   (make-validator
     keys #(and (not= ::absent %)
                (not (string/blank? %))
-               (or (not (try
-                          ;; TODO: better cljs impl
```
Comment (iku000888) [inline src/cljx/jkkramer/verily.cljx:138]: I feel a bit more comfortable leaving this comment here.

Comment (iku000888) [inline src/cljx/jkkramer/verily.cljx:138]: I feel a bit more comfortable leaving this comment here.

**Review by iku000888 — CHANGES_REQUESTED** [review]

**Comment by iku000888** [comment]
Thanks for the PR!
I presume this covers #10 ?
I will merge this once the comment is added back.

File: src/cljx/jkkramer/verily.cljx (hunk 1 above)
Comment (timothypratley) [reply src/cljx/jkkramer/verily.cljx:138]: okidoki, I've added it back in (see below)

**Comment by iku000888** [comment]
@timothypratley 
Made a release with the changes.
Would much appreciate if you can sanity check that your changes are included.
Thanks.

https://clojars.org/jkkramer/verily/versions/0.6.1

**Comment by timothypratley** [comment]
works great for me, thanks! (sorry for the slow reply)

**Comment by iku000888** [comment]
:+1: 
If you have any other enhancements, please do not hesitate to share :)

**Pull request merged**